### PR TITLE
`const` the other require() calls

### PR DESCRIPTION
**Pull request by jayphelps**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-var gulp = require('gulp');
-var $ = require('gulp-load-plugins')();
+const gulp = require('gulp');
+const $ = require('gulp-load-plugins')();
```
Comment (jmeas) [inline gulpfile.js]: Let's just leave these two as constants?

**Comment by jmeas** [comment]
Thanks for the PR, @jayphelps!

File: gulpfile.js (hunk 1 above)
Comment (jayphelps) [inline gulpfile.js]: Just to clarify--don't make any other variables constants except the requires?

Comment (jmeas) [inline gulpfile.js]: Yup – sorry, I could have worded that much more clearly. :stuck_out_tongue:

Comment (jayphelps) [inline gulpfile.js]: Done.

**Comment by jmeas** [comment]
Thanks again :+1:

**Pull request merged**